torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1033,6 +1033,28 @@ Example::
     >>> store.add("first_key", 6)
     >>> # Should return 7
     >>> store.get("first_key")
+)")
+          .def(
+              "check",
+              &::c10d::Store::check,
+              py::call_guard<py::gil_scoped_release>(),
+              R"(
+The call to check whether a given list of ``keys`` have value stored in
+the store. This call is nonblocking.
```

Comment:
it says its nonblocking.  But did you find a way to make it true? (if server does not respond, do we return early?)

Reply:
Well currently, I don't have a very strong proof to say it is 100% non-blocking. I think we have a plan to fix it. But to unblock the dump work, maybe I should just remove it?

Reply:
For example, by looking at HashStore, FileStore and TCPStore's code, aside from the mutex lock, there is no blocking logic, here.. Maybe, we should not say non-blocking for now